### ISSUE TITLE
Inconsistent G4 Doorbell Pro video via Apple TV

### ISSUE THREAD

**Issue by Krell53** [hjdhjd/homebridge-unifi-protect#973]
**Describe The Problem:**
With home bridge-unifi-protect installed on Homebridge and Homebridge added to my Apple HomeKit setup, when watching AppleTV and the doorbell is rung I would expect the video from the G4 Doorbell Pro to appear on the AppleTV stream to my TV. I have seen this happen once in the last year. When I look at the accessories tab in Homebridge there are numerous doorbell options that can be selected, most of which appear to be toggles, if I select one, then whatever I had selected prior is deselected. Since I did see the video at least once on AppleTV when the doorbell was rung, I assume I have the wrong accessories selected, or too many options configured in Homebridge. I have no idea what each of the accessories do? Do I only need to configure the main G4 Doorbell Pro and G4 Doorbell Pro Package options, and ignore all of the other video options to get the video popup on AppleTV to work? 

**To Reproduce:**
Setup the home bridge-unifi-protect using the web UI, add bridge to Apple HomeKit. Be watching AppleTV and have someone ring the doorbell, 99% of the time no video is displayed on the TV connected to AppleTV. I know I did see video appear at least once, so something worked, and since video from the doorbell did appear on the TV connected to the AppleTV I would assume that is expected behavior. I'd love to know what magic incantation I accidentally did the one time I saw the doorbell video on the TV.

**Logs:**

```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Configuration:**

```json
Show the relevant portion of your homebridge config.json here, if needed.
Remove any sensitive information.
```
{
    "controllers": [
        {
            "address": "Controller IP Address",
            "defaultDoorbellMessage": "Leave Package by Column",
            "doorbellMessages": [
                {
                    "duration": 0,
                    "message": "Please Ring Doorbell"
                }
            ],
            "name": "Krell UDM Pro SE",
            "password": my-password,
            "username": my-username
        }
    ],
    "name": "UniFi Protect",
    "options": [
        "Enable.Video.Stream.Only.Low",
        "Enable.Audio.Filter.Noise.AC8BA925D39D",
        "Enable.Audio.Filter.Noise.FftNr.AC8BA925D39D.90",
        "Enable.Audio.Filter.Noise.HighPass.AC8BA925D39D.200",
        "Enable.Audio.Filter.Noise.LowPass.AC8BA925D39D.1000",
        "Enable.Device.StatusLed.AC8BA925D39D",
        "Enable.Video.Transcode.Hardware.AC8BA925D39D",
        "Disable.Video.Stream.Only.Low.AC8BA925D39D",
        "Enable.Video.DynamicBitrate.AC8BA925D39D"
    ],
    "ringDuration": 10,
    "_bridge": {
        "username": my-username,
        "port": 48393
    },
    "platform": "UniFi Protect"
}

**Screenshots:**

![image](https://github.com/hjdhjd/homebridge-unifi-protect/assets/100861211/9dc3eb83-7f9a-4c3f-9aa3-ab0cfebae9ec)

**Environment:**

* **Homebridge Version:** v1.6.1
* **Node Version:** v18.16.0
* **Homebridge-UniFi-Protect Plugin Version:**Homebridge-unifi-protect v6.13.1
* **UniFi Protect Version:** v2.8.35
* **UniFi Protect Controller Type:**  UDM Pro SE
* **UniFi Protect Controller Firmware Version:** v3.1.15
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:**AppleTV v16.6
* **Operating System and OS Version:** 13.5 (22G74)

**Comment by hjdhjd** [comment]
Thanks for the report, however, this isn’t a bug or an issue in HBUP. You may want to reboot your HomeKit hubs - rings are processed by HomeKit and your Apple TV makes those decisions. If you see a ring event detected in the Homebridge logs, which you should, that ring event is passed on to HomeKit. Beyond that, HomeKit decides what to do.

TL;DR: reboot your Apple TV and other HomeKit hubs…this isn’t an HBUP issue I’m afraid. I’d also suggest using the Homebridge Discord and the unifi-protect channel - the community is helpful and can help you further if you’re having challenges.

Best of luck!
